commands.js

```diff
@@ -346,6 +390,7 @@ var commands = exports.commands = {
 
 	modjoin: function (target, room, user) {
 		if (!this.can('privateroom', null, room)) return;
+		if (room.isPersonal) return this.errorReply("Personal rooms configuration can't be changed.");
```

Comment:
`modjoin` should definitely be open to personal rooms. Basically any feature open to battle rooms should be open to personal rooms.